configs/laramie_bmi_config_ftm.txt

```diff
@@ -4,13 +4,16 @@ end_time_d=1035.91
 dt_s=3600.0
 soil_params.smcmax=0.439
 soil_params.b=5.25
-soil_params.psisat=0.355
+soil_params.satpsi=0.355
 soil_params.quartz=0.4
 surface_temperature_bc=260
-#ice_fraction_scheme=Schaake
-ice_fraction_scheme=BMI
+ice_fraction_scheme=Schaake
+#ice_fraction_scheme=BMI
 Z=0.1,0.5,1.0,2.0
 soil_temperature=280.15,280.15,280.15,280.15
-#soil_total_moisture_content=0.389,0.396,0.397,0.397
-#soil_liquid_moisture_content=0.389,0.396,0.397,0.397
+#Z=0.1,0.5,1.0,2.0,3.0,4.0,5.0,6.0
```

Comment:
Delete these commented out soil layer (Z) and soil temp (soil_temperature) options.  They clutter the config file.  I think it's ok to leave the commented out ice_fraction_scheme to show the different options until we have more detailed documentation with that information.

Reply:
deleted